src/openms/source/FORMAT/AbsoluteQuantitationMethodFile.cpp

```diff
@@ -86,17 +97,21 @@ namespace OpenMS
     {
       s.trim();
     }
-    aqm.setComponentName(tl[headers.at("component_name")]);
-    aqm.setFeatureName(tl[headers.at("feature_name")]);
-    aqm.setISName(tl[headers.at("IS_name")]);
-    aqm.setLLOD(tl[headers.at("llod")].empty() ? 0 : std::stod(tl[headers.at("llod")]));
-    aqm.setULOD(tl[headers.at("ulod")].empty() ? 0 : std::stod(tl[headers.at("ulod")]));
-    aqm.setLLOQ(tl[headers.at("lloq")].empty() ? 0 : std::stod(tl[headers.at("lloq")]));
-    aqm.setULOQ(tl[headers.at("uloq")].empty() ? 0 : std::stod(tl[headers.at("uloq")]));
-    aqm.setConcentrationUnits(tl[headers.at("concentration_units")]);
-    aqm.setNPoints(tl[headers.at("n_points")].empty() ? 0 : std::stoi(tl[headers.at("n_points")]));
-    aqm.setCorrelationCoefficient(tl[headers.at("correlation_coefficient")].empty() ? 0 : std::stod(tl[headers.at("correlation_coefficient")]));
-    aqm.setTransformationModel(tl[headers.at("transformation_model")]);
+    aqm.setComponentName(headers.count("component_name") ? tl[headers.at("component_name")] : "");
+    aqm.setFeatureName(headers.count("feature_name") ? tl[headers.at("feature_name")] : "");
+    aqm.setISName(headers.count("IS_name") ? tl[headers.at("IS_name")] : "");
+    aqm.setLLOD(!headers.count("llod") || tl[headers.at("llod")].empty() ? 0 : std::stod(tl[headers.at("llod")]));
```

Comment:
OK this is the behavior the original code was trying to capture...except in 1 line instead of 5...which is nice.